### DISCUSSION TITLE
Downgrade to v3.4.2?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [cross-seed/cross-seed#267] [Q&A] [answered]
Pardon this kind of general newb-ish question. Is there a way to go back to version 3.4.2? I'd like to be able to test out v4+ but have written sort of elaborate (to me) AppleScripts to work with my current setup, and I'd like to be able to downgrade if I run into a problem.

**Comment by mmgoodnow** [comment, marked as answer]
You can always install a different version with npm. I'm not sure of the syntax but the npm docs should say

**Reply by SawkeeReemo** [reply]
ok, thanks! I'll look into it.
